zookeeper-client/zookeeper-client-c/README

```diff
@@ -163,3 +163,23 @@ In order to be able to use the zookeeper API in your application you have to
 
 Please take a look at cli.c to understand how to use the two API types. 
 (TODO: some kind of short tutorial would be helpful, I guess)
+
+How to build your own project in Windows or Linux use the c api:
+For example :
+we use the cmakefile to build project
+
+   IF (WIN32)
+      MESSAGE(STATUS "Now is windows")
+      add_definitions(-DUSE_STATIC_LIB -DTHREADED)
+      link_libraries(zookeeper.lib)
+      target_link_libraries(test_zookeeper_c zookeeper.lib hashtable.lib ws2_32.lib)
```

Comment:
Why do we have a `link_libraries` *and* a `target_link_libraries`, with different arguments?

I belive we should recommend one *or* the other.  `target_link_libraries` is "safer," but more complicated as the first parameter (`test_zookeeper_c` in your example) has to be adapted by the user.

Reply:
The list of dependencies actually depend on the compilation options; OpenSSL and SASL are two important optional dependencies.  (They will often be automatically picked up by Autoconf under Linux-based OSes.)